kovri.i2p/_i18n/es/acknowledgements.md

```diff
@@ -1,9 +1,9 @@
-En adición a los contribuidores de Boost y OpenSSL, y a todos los programadores desde los tiempos inmemoriales, que han hecho este proyecto posible, queremos reconocer lo siguiente:
+En adición a los contribuidores de Boost y OpenSSL y a todos los programadores desde los tiempos inmemoriales que han hecho este proyecto posible, queremos reconocer a los siguientes:
 
-- Wei Dai, Jeffrey Walton, y todos los desarrolladores en Cryoto++ por proveer una influencial y libre librería de criptografía en C++
-- Dean Michael Berris, Glyn Matthews, y todos los desarrolladores en cpp-netlib por proveer una librería activamente desarrollada cross-plataforma de red.
+- Wei Dai, Jeffrey Walton y todos los desarrolladores de Crypto++ por proveer una influencial y libre librería de criptografía en C++
+- Dean Michael Berris, Glyn Matthews y todos los desarrolladores de cpp-netlib por proveer una librería activamente desarrollada cross-plataforma de red.
```

Comment:
Dean Michael Berris, Glyn Matthews, and all the developers at cpp-netlib for providing an actively developed cross-platform networking library.

Esta sería mi traducción:
"A Dean Micheal Berris, Glyn Matthews y todos los desarrolladores de cpp-netlib por proveer una biblioteca de redes **(o interconectada)** multiplataforma **(o de plataforma cruzada)** activa.